frontend/src/store/modules/place/actions.js

```diff
@@ -44,4 +44,52 @@ export default {
                 });
         });
     },
+
+    likePlace: (context, placeId) => {
+        return new Promise((resolve, reject) => {
+            httpService.post('places/' + placeId + '/like')
+                .then(function (res) {
+                    if (res.status === 400){
```

Comment:
remove `if (res.status === 400){...}` this will go to the catch block. Also result will be the same
```
return httpService.post('places/' + placeId + '/like')
    .then((res) =>{
        resolve(res.data);
    });
```